### ISSUE TITLE
CLI tool outputs incorrect results on Ubuntu 20

### ISSUE THREAD

**Issue by AlexElin** [stylelint/stylelint#6397]
### What steps are needed to reproduce the bug?

Run stylelint for the [project](https://stackblitz.com/edit/angular-2rgmhu?file=src%2F.stylelintrc.json,.stackblitzrc) via CLI on Windows 10 and Ubuntu 20.
Then compare results.

Instead of Ubuntu 20, I've received similar results while running the checks on GitLab CI having the below configuration. 

```yml
stages:
  - check
  
variables:
  NODE_IMAGE: node:16.13.1
  ANGULAR_CLI_VERSION: 14.0.5
  
check-style:
  stage: check
  image: $NODE_IMAGE
  before_script:
    - npm install -g @angular/cli@"$ANGULAR_CLI_VERSION"
  script:
    - npm ci
    - find ./src -name "*.css"
    - npx stylelint -f verbose src/**/*.css
  only:
    - merge_requests

```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard"
}

```

### How did you run Stylelint?

npx stylelint -f verbose src/**/*.css

### Which version of Stylelint are you using?

14.13.0

### What did you expect to happen?

Results match on different OSs

### What actually happened?

Results of the CLI command were different

1. Ubuntu 20
6 errors were detected
![image](https://user-images.githubusercontent.com/13061803/194710925-9c675b44-cc8d-466f-995d-729642e53a91.png)
2. Windows 10
76 errors were detected
![image](https://user-images.githubusercontent.com/13061803/194710912-2d98f06f-2d1c-4abf-a481-6f5255491a1d.png)
skipping some part
![image](https://user-images.githubusercontent.com/13061803/194710902-6f1d8532-c631-4903-9abd-bc9e6ea1dbdd.png)

**Comment by AlexElin** [comment]
@alexander-akait thanks. 
It helped me
